transcom/mymove
# B-18648 Update UI to reflect the changes made.

## [B-18648](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18648) 

# Requested Changes
- Weight-based distance multiplier:
Move from the bottom of the page to the Billable weight area (under Estimated weight)
Move value to the right
Weight based multiplier needs to show up to 7 digits after the decimal

- Mileage
Add thousands separator comma (i.e. 1,664 not 1664)

- Fuel Surcharge Price
Update label to read Mileage Factor
Update value to show up to 3 digits past the decimal

- Pickup Date
Move Pickup date above EIA diesel

- EIA diesel value
should extend to 3 digits and not round to 2 (i.e. $3.876 not $3.88)

- Fuel Rate Adjustment (FRA)
Update label to read Baseline Rate Difference
Move value to the right column

- Total Amount Requested
Update label to read Fuel Rate Adjustment

## Steps I used to test
1. Use the "HHGMoveInSITEndsToday" option from the Test harness list.
2. Copy the move locator from the JSON object displayed.
3. Navigate back to the sign in page for milmove office users
4. Sign in as a Multi-role user under the KKFA GBLOC.
5. Start by clicking the Sign in button for a Prime user.
6. Search for the move by the move locator you copied earlier. Then click on that move in the list.
7. Click the "Create payment request" button.
8. From the list of service items you are able to select from you should see an item called "Domestic origin SIT fuel surcharge". Put a check in the checkbox for that item ("Add to payment request").
9. Click the "Submit payment request" button at the bottom of the form.
10. Click the "Change user role" link at the top of the page.
11. Make the appropriate selection to log in as a TOO user.
12. Search for the same move locator and click on the associated move.
13. Click the "Payment requests" tab.
Note: there is a known issue when certain conditions are met that the TOO must review weights associated. Steps 15-16 are to workaround this known issue.
14. Click the "Review Weights" button.
15. Click the "Payment requests" tab again.
16. Click the "Review Service Items" button.
17. Observe that the service item called "Domestic origin SIT fuel surcharge" has a "Show calculations" link. Click that link to show the changes listed above.
